lib/runtime2/threepio/capabilities/generate-domcument/parsers/interface.ts

```diff
@@ -1,6 +1,37 @@
 import { ApiStreamTextChunk } from "../../../api/transform/stream";
+import { EmitData, MoudleParserEventType, ParsedHeader, ParsedModule } from "../interfaces";
 
+/**
+ * Stream for generating HTML fragments.
+ */
+export type HtmlGenerateStream = AsyncGenerator<HtmlStreamItem, void, unknown>;
```

Comment:
```suggestion
export type HTMLGeneratingStream = AsyncGenerator<HTMLStreamItem, void, unknown>;
```
同理：

1. HTML 是专有名词
2. Generate 是动词，没发作为形容词修饰 Stream